src/Expensify.js

```diff
@@ -127,9 +127,9 @@ class Expensify extends PureComponent {
             const shouldHideSplash = !this.isAuthenticated() || authStackReady;
 
             if (shouldHideSplash) {
-                BootSplash
-                    .hide({fade: true})
-                    .then(() => this.setState({isSplashShown: false}));
```

Comment:
Just wanted to check that we no longer want to await the completion of `hide` before updating the state?

Reply:
Currently, awaiting the `hide` will only update the state once the fadeout is complete. I think it's better to continue immediately has there is no issue with it anymore (no more clashes between the splash screen and native modals / dialogs).

Reply:
Thanks, that makes sense.